addon/templates/components/uni-progress-bar.hbs

```diff
@@ -1 +1,5 @@
 <div role="progressbar" class="uni-progress-bar__bar" style={{inlineStyle}}></div>
+
+{{#x-times totalStepsCount as |step|}}
+  <div class="uni-progress-bar__step" style="left: calc({{eachStepPercentage}}% * {{step}});"></div>
```

Comment:
Extract the style to a computed property and use `htmlSafe`. This should have emitted a warning